mvcc/kvstore.go

```diff
@@ -164,6 +164,7 @@ func (s *store) compactBarrier(ctx context.Context, ch chan struct{}) {
 	close(ch)
 }
 
+// TODO: hash and revision could be inconsistent, one possible fix is to add s.revMu.RLock() at the beginning of function, which is costly
```

Comment:
https://travis-ci.com/etcd-io/etcd/jobs/229792898#L564

goword is suggesting to use 'Hash'.